tajo-algebra/src/main/java/org/apache/tajo/algebra/AlterTable.java

```diff
@@ -43,6 +44,8 @@
   private AlterTableOpType alterTableOpType;
   @Expose @SerializedName("TableProperties")
   private Map<String, String> params;
+  @Expose @SerializedName("TablePropertyKeys")
+  private List<String> propertyKeys;
```

Comment:
It would be better to change this variable to be more intuitive as well. I think getter and setter also need to be changed together.